io.github.giantpinkrobots.flatsweep.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "io.github.giantpinkrobots.flatsweep",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flatsweep",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
this is not allowed by default

https://docs.flathub.org/docs/for-app-authors/linter

Reply:
I added this line because I needed to get the list of installed Flatpaks on the system and the way to do that was to run a flatpak list command using flatpak-spawn. If that is not allowed I will try to get the list using some other way. But that is the reason why.

Reply:
If you have a good reason, you can add a exception

Reply:
You could add RO access to /var/lib/flatpak/app and xdg-data/flatpak/app. The directories there are the IDs of currently installed apps.

Reply:
This is not enough. You can add a [custom installation path](https://docs.flatpak.org/en/latest/tips-and-tricks.html#adding-a-custom-installation). This won't be done by 99% of the users, but who those who do it, it's better to have it working.

Reply:
I wouldn't particularly care about less than 1% of users using custom installation paths.

Reply:
Is there a way to check if the user has chosen an alternative install path so I could show an error screen? Without using commands.